src/app/[lang]/app/profile/[id]/page.tsx

```diff
@@ -46,7 +46,7 @@ export default async function Profile({ params }: PageParams<{ id: string }>) {
         </ProfileSectionLayout>
         <ProfileSectionLayout title='Services' id='profile-services'>
           <Suspense fallback={<ServiceTableSkeleton rows={3} />}>
-            <UserServices userId={params.id} />
+            <UserServices userId={params.id} session={session} />
```

Comment:
I don't think it's a good idea to pass session as a props, can't we get the session inside `UserServices`?

Reply:
From Vadim:  We can use useSession but it works slowly if compared to getServerSession on top of the page. For now, I don’t see better solutions

Reply:
I see, okay then, let's leave it like that for now.